js/noms/src/sequence.js

```diff
@@ -126,10 +126,11 @@ export class SequenceCursor<T, S: Sequence<any>> {
   getChildSequence(): Promise<?S> {
     if (this.readAhead && this.sequence.isMeta && !this.childSeqs) {
       // Only readAhead when enabled, for meta sequences.
-      this.childSeqs = [];
+      const childSeqs = [];
```

Comment:
Need this temporary variable because this change makes flow sad without it ("childSeqs may be null or undefined"). I don't know why it didn't have this warning before.